src/app/components/media/ReportDesigner/ReportDesignerForm.js

```diff
@@ -90,292 +55,490 @@ const ReportDesignerForm = (props) => {
+                  description="Placeholder for report url field"
+                >
+                  { placeholder => (
+                    <LimitedTextArea
+                      required={Boolean(false)}
+                      value={data.published_article_url || ''}
+                      componentProps={{
+                        'data-testid': 'report-designer__text-url',
+                      }}
+                      maxChars={140}
+                      maxlength="140"
+                      key={`text-url-${data.language}-${data.published_article_url}`}
+                      rows={1}
+                      label={
+                        <FormattedMessage
+                          id="reportDesigner.textUrl"
+                          defaultMessage="Article URL"
+                          description="Text report field"
+                        />
+                      }
+                      disabled={props.pending || props.disabled}
+                      maxHeight="48px"
+                      autoGrow="false"
+                      placeholder={placeholder}
+                      onBlur={(e) => {
+                        const newValue = e.target.value;
+                        let newUrl = newValue;
+                        if (newValue.trim().length !== 0 && !/^https?:\/\//.test(newUrl)) {
+                          newUrl = `https://${newUrl}`;
+                        }
+                        props.onUpdate('published_article_url', newUrl);
+                      }}
+                    />
+                  )}
+                </FormattedMessage>
+              </div>
+            </> : null
+          }
+          { data.use_visual_card && !data.use_text_message ?
+            <>
+              <ToggleButtonGroup
+                key={`dark-overlay-${data.language}`}
+                label={<FormattedMessage id="reportDesigner.theme" defaultMessage="Theme" description="Label used for choice on theme of report light versus dark" />}
+                className={inputStyles['form-fieldset-field']}
+                variant="contained"
+                onChange={(e, newValue) => {
+                  props.onUpdate('dark_overlay', Boolean(newValue));
+                }}
+                exclusive
+              >
+                <ToggleButton
+                  selected={Boolean(!data.dark_overlay)}
+                  value={Boolean(false)}
+                  key="false"
+                  className="int-report__button--report-type"
+                >
+                  <FormattedMessage id="reportDesigner.lightTheme" defaultMessage="Light" description="Label used for button toggling on the light themed report" />
+                </ToggleButton>
+                <ToggleButton
+                  selected={Boolean(data.dark_overlay)}
+                  value={Boolean(true)}
+                  key="true"
+                  className="int-report__button--report-type"
```

Comment:
It looks like these classNames are repeated. If we're using them for automated tests, how about changing them to something like `int-report__button--theme-dark` and `int-report__button--theme-light` ?